services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/api/containers_extension.py

```diff
@@ -132,7 +132,7 @@ async def pull_input_ports(
     response_class=Response,
     status_code=status.HTTP_204_NO_CONTENT,
 )
-async def disable_directory_watcher(
+async def enabled_disable_directory_watcher(
```

Comment:
this read very weird since it contains "enabled" and "disable" in the same name! I guess you mean ``enable_directory_watcher`` or ``toggle_directory_watching``?

Reply:
yes, those look better!